src/igvfd/upgrade/page.py

```diff
@@ -10,3 +10,9 @@ def page_1_2(value, system):
     if 'aliases' in value:
         if len(value['aliases']) == 0:
             del value['aliases']
+
+
+@upgrade_step('page', '2', '3')
```

Comment:
I think the ugrade code has to drop the potentially present properties of Page object. Then the test of upgrade will need to check that these properties were actually dropped by the upgrade